src/Features/Lsif/Generator/Generator.cs

```diff
@@ -400,6 +411,30 @@ void MarkImplementationOfSymbol(ISymbol baseMember)
             }
         }
 
+        private static (DefinitionRangeTag tag, TextSpan fullRange)? CreateRangeTagAndContainingSpanForDeclaredSymbol(ISymbol declaredSymbol, SyntaxToken syntaxToken, SyntaxTree syntaxTree, ISyntaxFacts syntaxFacts, CancellationToken cancellationToken)
+        {
+            // Find the syntax node that declared the symbol in the tree we're processing
+            var syntaxReference = declaredSymbol.DeclaringSyntaxReferences.FirstOrDefault(static (r, syntaxTree) => r.SyntaxTree == syntaxTree, arg: syntaxTree);
+            var syntaxNode = syntaxReference?.GetSyntax(cancellationToken);
+
+            if (syntaxNode is null)
+                return null;
+
+            // The containing range is supposed to be "the full range of the declaration not including leading/trailing whitespace, but everything else", so produce our start/end points
```

Comment:
note: i'm still not sure if this means comments or not.  the text itself isn't very clear since it could be the case that comments are considered part of a declaration ever.  But i think it's fine to start with this.

Reply:
Ah I guess I chopped off the full from the spec:

```
  /**
   * The full range of the definition not including leading/trailing whitespace
   * but everything else, e.g comments and code. The range must be included in
   * fullRange.
   */
```

I'll copy the rest of that in for clarity.